features/create_articles.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Create articles
+  As a journalist
+  in order to attract subscribers
+  I would like to be able to create articles
+
+  Background:
+    Given I visit the "index" page
+    When I click "New Article" link
+
```

Comment:
Unnessesary use of Background, move these steps to Scenario. And change "When " to "And"